app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,4 @@
 <resources>
     <string name="app_name">imhungry</string>
+    <string name="settings">Settings</string>
```

Comment:
This should be more descriptive. Once we get a lot more strings in the app, ambiguous strings get annoying.

I recommend `action_bar_settings_label` or something similar.